src/css/handsontable.css

```diff
@@ -585,21 +585,48 @@ CheckboxRenderer
   background: #eee;
 }
 
-.ht_clone_top {
-  z-index: 101;
+.ht_editor_hide {
+  z-index: -1;
+}
+
+.ht_editor_show {
```

Comment:
I would call this class as `ht_editor_shown`.

Reply:
Maybe "visible" would be better?